prowler/providers/aws/services/acm/acm_certificates_with_secure_key_algorithms/acm_certificates_with_secure_key_algorithms.metadata.json

```
@@ -1,31 +1,46 @@
+    "Software and Configuration Checks/Industry and Regulatory Standards/NIST 800-53 Controls (USA)"
   ],
   "ServiceName": "acm",
   "SubServiceName": "",
-  "ResourceIdTemplate": "arn:partition:acm:region:account-id:certificate/resource-id",
+  "ResourceIdTemplate": "",
   "Severity": "high",
   "ResourceType": "AwsCertificateManagerCertificate",
-  "Description": "Check if ACM Certificates use a secure key algorithm (RSA 2048 bits or more, or ECDSA 256 bits or more). For example certificates that use RSA-1024 can be compromised because the encryption could be broken in no more than 2^80 guesses making it vulnerable to a factorization attack.",
-  "Risk": "Certificates with weak RSA or ECDSA keys can be compromised because the length of the key defines the security of the encryption. The number of bits in the key determines the number of guesses an attacker would have to make in order to decrypt the data. The more bits in the key, the more secure the encryption.",
-  "RelatedUrl": "https://docs.aws.amazon.com/acm/latest/userguide/acm-certificate.html",
+  "Description": "**ACM certificates** are evaluated for the **public key algorithm and size**, identifying those that use weak parameters such as `RSA-1024` or ECDSA `P-192`. Certificates using `RSA-2048+` or ECDSA `P-256+` meet the secure baseline.",
+  "Risk": "**Weak certificate keys** reduce TLS confidentiality and authenticity.\n\nFeasible factoring or discrete log attacks can reveal private keys, enabling **man-in-the-middle**, session decryption, and **certificate spoofing**, leading to data exposure and tampering.",
+  "RelatedUrl": "",
+  "AdditionalURLs": [
+    "https://noise.getoto.net/2022/11/08/how-to-evaluate-and-use-ecdsa-certificates-in-aws-certificate-manager/",
+    "https://registry.terraform.io/modules/tedilabs/domain/aws/latest/submodules/amazon-issued-cert",
+    "https://www.encryptionconsulting.com/difference-between-various-public-key-infrastructure/",
+    "https://stackoverflow.com/questions/67606544/how-to-solve-aws-cloudfront-ssl-certificate-doesnt-exist",
+    "https://repost.aws/questions/QUhtmyY4d5SK6dvKUen8A7xg/aws-acm-algorithm-and-encryption",
+    "https://sdk.amazonaws.com/kotlin/api/latest/acm/aws.sdk.kotlin.services.acm.model/-request-certificate-request/-builder/key-algorithm.html",
+    "https://docs.aws.amazon.com/acm/latest/userguide/acm-certificate.html",
+    "https://docs.aws.amazon.com/securityhub/latest/userguide/acm-controls.html#acm-2"
+  ],
   "Remediation": {
     "Code": {
       "CLI": "",
-      "NativeIaC": "",
-      "Other": "",
-      "Terraform": ""
+      "NativeIaC": "```yaml\n# CloudFormation: ACM certificate with secure key algorithm\nResources:\n  <example_resource_name>:\n    Type: AWS::CertificateManager::Certificate\n    Properties:\n      DomainName: <example_domain>\n      KeyAlgorithm: EC_prime256v1  # CRITICAL: ensures a secure key algorithm (ECDSA P-256) to pass the check\n```",
```

```suggestion
      "NativeIaC": "```yaml\n# CloudFormation: ACM certificate with secure key algorithm\nResources:\n  <example_resource_name>:\n    Type: AWS::CertificateManager::Certificate\n    Properties:\n      DomainName: <example_domain>\n      KeyAlgorithm: EC_prime256v1  # CRITICAL: ensures a secure key algorithm (RSA-2048+ or ECDSA P-256+)\n```",
```